pkg/cloudprovider/fake/instancetype.go

```diff
@@ -44,33 +51,33 @@ func NewInstanceType(options InstanceTypeOptions) *InstanceType {
 	}
 	return &InstanceType{
 		InstanceTypeOptions: InstanceTypeOptions{
-			name:             options.name,
-			zones:            options.zones,
-			capacityTypes:    options.capacityTypes,
-			architecture:     options.architecture,
-			operatingSystems: options.operatingSystems,
-			cpu:              options.cpu,
-			memory:           options.memory,
-			pods:             options.pods,
-			nvidiaGPUs:       options.nvidiaGPUs,
-			amdGPUs:          options.amdGPUs,
-			awsNeurons:       options.awsNeurons,
+			name:          options.name,
+			zones:         options.zones,
+			capacityTypes: options.capacityTypes,
+			offerings:     options.offerings,
+			architecture:  options.architecture,
+			cpu:           options.cpu,
+			memory:        options.memory,
+			pods:          options.pods,
+			nvidiaGPUs:    options.nvidiaGPUs,
+			amdGPUs:       options.amdGPUs,
+			awsNeurons:    options.awsNeurons,
 		},
 	}
 }
 
 type InstanceTypeOptions struct {
-	name             string
-	zones            sets.String
-	capacityTypes    sets.String
-	architecture     string
-	operatingSystems sets.String
-	cpu              resource.Quantity
-	memory           resource.Quantity
-	pods             resource.Quantity
-	nvidiaGPUs       resource.Quantity
-	amdGPUs          resource.Quantity
-	awsNeurons       resource.Quantity
+	name          string
+	zones         sets.String
```

Comment:
possible to remove zone/capacityTypes?

Reply:
Darn it, in the midst of test debugging I missed this - thanks!